zzzz_modular_occulus/code/modules/research/nodes/combat.dm

```diff
@@ -20,6 +20,21 @@
 							/datum/design/research/item/holochip/tough,
 							/datum/design/research/item/holochip/painkiller)
 
+/datum/technology/advanced_armor
+	name = "Advanced Armor Solutions"
+	desc = "Advanced voidsuit with combined ballistic and ablative plating, and other personal protection."
+	tech_type = RESEARCH_COMBAT
+
+	x = 0.8
+	y = 0.5
+	icon = "moebiushelm"
```

Comment:
We may want to look into getting a unique sprite for this and our other unique techs. It's not a blocker, just a comment here.